File: Heat/ViewModels/Heaters/CreateHeaterViewModel.vb
Imports System.ComponentModel.DataAnnotations

Namespace ViewModels.Heaters
    Public Class CreateHeaterViewModel

        Property ThermalUnitID As Integer
        <Display(name:="Generatore di riferimento")> _
        Property ThermalUnitDescription As String

        <Display(name:="Marca")> _
        Property ManifacturerID As String
        Property ManifacturerList As IEnumerable(Of SelectListItem)

        <Display(name:="Modello")> _
        Property ModelID As Integer
        Property ModelList As IEnumerable(Of SelectListItem)

        <Display(name:="Matricola")> _
        Property SerialNumber As String

        <Display(name:="Portata termica minima nominale (kW)")> _
        Property MinimumPowerKW As Single

        <Display(name:="Portata termica massima nominale (kW)")> _
        Property MaximumPowerKW As Single

        <Display(name:="Tipologia")> _
        Property Type As String

        <Display(name:="Data di installazione")> _
        Property InstallationDate As DateTime

        <Display(name:="Data di dismissione")> _
        Property DismissDate As Nullable(Of DateTime)

        <Display(name:="Combustibile")> _
        Property FuelID As Integer
        Property FuelList As IEnumerable(Of SelectListItem)

        
    End Class

End Namespace

File: Heat.ConvertedToC#/ViewModels/Heaters/CreateHeaterViewModel.cs
using System;
using System.Collections.Generic;
using System.Web.Mvc;
using System.ComponentModel.DataAnnotations;

namespace Heat.ViewModels.Heaters
{
    public class CreateHeaterViewModel
	{

		public int ThermalUnitID { get; set; }
		[Display(Name = "Generatore di riferimento")]
		public string ThermalUnitDescription { get; set; }

		[Display(Name = "Marca")]
		public string ManifacturerID { get; set; }
		public IEnumerable<SelectListItem> ManifacturerList { get; set; }

		[Display(Name = "Modello")]
		public int ModelID { get; set; }
		public IEnumerable<SelectListItem> ModelList { get; set; }

		[Display(Name = "Matricola")]
		public string SerialNumber { get; set; }

		[Display(Name = "Portata termica minima nominale (kW)")]
		public float MinimumPowerKW { get; set; }

		[Display(Name = "Portata termica massima nominale (kW)")]
		public float MaximumPowerKW { get; set; }

		[Display(Name = "Tipologia")]
		public string Type { get; set; }

		[Display(Name = "Data di installazione")]
		public DateTime InstallationDate { get; set; }

		[Display(Name = "Data di dismissione")]
		public Nullable<DateTime> DismissDate { get; set; }

		[Display(Name = "Combustibile")]
		public int FuelID { get; set; }
		public IEnumerable<SelectListItem> FuelList { get; set; }


	}

}
